src/main/java/generated/LolChampionsCollectionsChampionMinimal.java

```diff
@@ -1,7 +1,16 @@
 package generated;
 
+import java.math.BigInteger;
 import java.util.List;
 
+/**
```

Comment:
This comment will get removed once I rerun the generation script. As for BigInteger change, I will incorporate that into the generation script. I might add generating comments like these as well, but I'm not sure yet. Feel free to leave it, but just wanted to give you a heads up, that it might disappear in the future.

Reply:
Do you want me to add it directly to the script? or do you think it is better to just ignore this section for now?

Reply:
For now let's leave it in. The script is in need of refactor anyways.